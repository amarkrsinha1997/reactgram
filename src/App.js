import React from 'react';
import logo from './logo.svg';
import './App.css';
import { Tab } from './Component/Base/Tabs/Tab';
import { Tabs } from './Component/Base/Tabs/Tabs';

function App() {
  return (
    <div className="App">
     <Tabs>
       <Tab> 
        Like 1
       </Tab>
       <Tab> 
        Like 1
       </Tab>
     </Tabs>
    </div>
  );
}

export default App;
