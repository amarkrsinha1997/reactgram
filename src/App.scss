@mixin lowMobile { 
	@media (min-width: 0px) and (max-width:319px) { @content } 
};
@mixin mobile { 
	@media (min-width: 320px) and (max-width:719px) { @content } 
};
@mixin mobileToPhablet { 
	@media (min-width: 320px) and (max-width: 1023px) { @content } 
};
@mixin mobileToTablet { 
	@media  (min-width: 320px) and (max-width: 1279px) { @content } 
};
@mixin mobileToDesktop { 
	@media  (min-width: 320px) { @content } 
};
@mixin phablet { 
	@media  (min-width: 720px) and (max-width: 1023px) { @content } 
};
@mixin phabletToTablet { 
	@media  (min-width: 720px) and (max-width: 1279px) { @content } 
};
@mixin phabletToDesktop { 
	@media  (min-width: 720px) { @content } 
};
@mixin tabletOnly { 
	@media  (min-width: 1024px) and (max-width: 1279px) { @content } 
};
@mixin tabletToDesktop { 
	@media  (min-width: 1024px) { @content } 
};
@mixin desktop { 
	@media  (min-width: 1280px) { @content } 
}

.App {
	display: flex;
	align-items: center;
	justify-content: center;
	flex-wrap: nowrap;
	flex-direction: column;
}

.headline {
  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
  color: blueviolet;
  font-style: italic;
  font-size: 54px;
}