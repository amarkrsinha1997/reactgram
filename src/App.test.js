import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

jest.mock('./data/posts', () => {
  return [	{
		"code": "_A2r0aQcfD",
		"caption": "Some serious hardware meet JavaScript hacks going down this week at hackeryou. Excited for demo day!",
		"likes": 57,
		"id": "1135147611821557699",
		"display_src": "12276809_750065668431999_184252508_n.jpg",
		"comment": [
			{
				"text": "Uhu!",
				"user": "lucascaixeta",
				"id": 1
			}
		]
	}]
})

test('renders learn react link', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
