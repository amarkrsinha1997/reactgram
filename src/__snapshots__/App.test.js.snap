// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders learn react link 1`] = `
<div>
  <div
    class="App"
  >
    <h1
      class="headline"
    >
      ReactGram
    </h1>
    <div
      class="post"
    >
      <div
        class="post-image"
      >
        <img
          alt="12276809_750065668431999_184252508_n.jpg"
          class="image"
          data-testid="12276809_750065668431999_184252508_n.jpg"
          src="https://raw.githubusercontent.com/pankajladhar/Reactgram/master/app/images/12276809_750065668431999_184252508_n.jpg"
        />
      </div>
      <div
        class="tabs"
      >
        <div
          class="seperator"
        >
          <button
            class="tab"
          >
            Like
          </button>
        </div>
        <div
          class="seperator"
        >
          <button
            class="tab"
          >
            Comments 
            (1)
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
